packages/datetime/src/common/dateUtils.ts

```diff
@@ -309,3 +310,20 @@ export function stringToMoment(dateString: string, format: DateFormat, locale: s
         return date === undefined ? moment.invalid() : moment(date);
     }
 }
+
+
+export function getLocale(localeString: String): any {
+    if(!localeString) {
+        return undefined;
+    }
+    return require(`date-fns/locale/${localeString}`);
```

Comment:
⚠️ probably can't ship this dynamic `require` statement... 
thoughts @adidahiya?